pymc/tests/test_distributions.py

```diff
@@ -2673,21 +2678,21 @@ def test_continuous(self):
         assert logpt(InfBoundedNormal, 0).eval() != -np.inf
         assert logpt(InfBoundedNormal, 11).eval() != -np.inf
 
-        assert logpt(LowerNormalTransform, -1).eval() != -np.inf
+        # TODO: Investigate evaluation issue in aeppl over here.
+        # This passed since passed value to aeppl is simply a TensorConstant
```

Comment:
ToDo (convert to issue?)

Reply:
Maybe. Otherwise the better way to do this would probably be to use a corresponding value variable while constructing the log probability graph.